starknet_devnet/util.py

```diff
@@ -123,6 +123,11 @@ def parse_args():
         help=f"Specify when to dump; can dump on: {DUMP_ON_OPTIONS_STRINGIFIED}",
         type=parse_dump_on
     )
+    parser.add_argument(
+        "--lite-mode",
```

Comment:
How much work would it be to split this into multiple options? Like allowing for a more configurable approach with a separate `--blocks` and `--tx_hash` options. E.g. `--blocks lite`, `--blocks full`, `--tx_hash lite`, `--tx_hash real`?

Reply:
what about something like `--lite-mode-blocks` and `--lite-mode-tx-hash`, if a flag is passed the lite mode for it will be enabled